src/parser/view_transform.c

```diff
@@ -5335,6 +5336,13 @@ mq_translate_insert (PARSER_CONTEXT * parser, PT_NODE * insert_statement)
 	  flat = from->info.spec.flat_entity_list;
 	  if (flat == NULL)
 	    {
+	      if (from_spec->remote_server_name)
```

Comment:
Isn't that unnecessary code?

Reply:
the spec can be remote-server under the flat_entity_list is NULL.